### ISSUE TITLE
Review Accessibility 

### ISSUE THREAD

**Issue by artuoma** [cagov/covid19#288]
### Test for and identify any accessibility issues
- [x] Homepage
- [x] Improve tap targets as outlines in SEO report
- [x] Top pages
- [x] All remaining pages

*This should be re-done after the redesign
